Cargo.toml

```diff
@@ -22,6 +22,6 @@ codegen-units = 1
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
 bellman = { git = "https://github.com/zkcrypto/bellman.git", rev = "29c82c7e432c1b32a50ef58e42790c3963ef2b4b" }
-orchard = { git = "https://github.com/zcash/orchard.git", rev = "61a67f018f456da980de7f93a5ea2a1dc5a61e63" }
-halo2_gadgets = { git = "https://github.com/zcash/halo2.git", rev = "35eb268e202d76721abde3d6e50e719ae9140605" }
-halo2_proofs = { git = "https://github.com/zcash/halo2.git", rev = "35eb268e202d76721abde3d6e50e719ae9140605" }
+orchard = { git = "https://github.com/zcash/orchard.git", rev = "6cbde279e90974201bedbd9b5ddf155e8f8b1e8e" }
+halo2_gadgets = { git = "https://github.com/zcash/halo2.git", rev = "642924d614305d882cc122739c59144109f4bd3f" }
+halo2_proofs = { git = "https://github.com/zcash/halo2.git", rev = "642924d614305d882cc122739c59144109f4bd3f" }
```

Comment:
I verified that this is the current revision of the `main` branch for `zcash/halo2`.